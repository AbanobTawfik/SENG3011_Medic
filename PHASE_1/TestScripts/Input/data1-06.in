#!/bin/sh
# requested date range overlaps with beginning of event date range

start_date=2010-03-15T00:00:00
end_date=2010-03-25T00:00:00
