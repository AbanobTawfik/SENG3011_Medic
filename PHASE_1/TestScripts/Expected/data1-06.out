{
    "meta": {
        "team_name": "Medics",
        "accessed_time": "REMOVED",
        "data_source": {
            "name": "CDC",
            "url": "https://www.cdc.gov/outbreaks/"
        }
    },
    "articles": [
        {
            "url": "url2",
            "reports": [
                {
                    "diseases": [],
                    "syndromes": [],
                    "event_date": "2010-03-20 xx:xx:xx to 2010-04-03 xx:xx:xx",
                    "locations": []
                }
            ],
            "headline": "Headline of url2",
            "main_text": "This is the main text for url2.",
            "date_of_publication": "2010-04-05 12:00:00"
        }
    ]
}
